backend/repository/endpoint_update_test.go

```diff
@@ -57,8 +58,13 @@ func testUpdateEndpointConfig(t *testing.T, tempDir string, requestFile string)
 	err := readJSONFile(requestFile, req)
 	assert.NoError(t, err, "Failed to unmarshal endpoint config.")
 	r := &Repository{
-		localDir: tempDir,
+		localDir:        tempDir,
+		refreshInterval: time.Hour * 65535,
```

Comment:
what is this magic number `65535`?

Reply:
no magic # :) a number large enough so we don't refresh repo during a test